apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -66,12 +66,18 @@ const style = {
     flexGrow: 1,
     marginBottom: 0,
     boxShadow: 'none'
+  },
+  inspector: {
+    display: 'inline-block'
   }
 };
 
 const WATCH_COMMAND_PREFIX = '$watch ';
 const UNWATCH_COMMAND_PREFIX = '$unwatch ';
 
+// did not include undefined since we do not want to
+// return undefined when input is console.log("something")
+const FALSIES = new Set([false, null, 0, '', ``, NaN]);
```

Comment:
NIT: Suggested name: FALSY or FALSY_VALUES

Reply:
1. Working on making that case of console.log(undefined) work!
2. Removed ``! 
3. Changed to FALSY_VALUES! 
4. Added suggested comment!